aptoide-ui/src/main/java/cm/aptoide/pt/aptoide_ui/theme/Color.kt

```diff
@@ -21,3 +21,8 @@ val purpleCatappult = Color(0xFF190054)
 val appCoins = Color(0xFFFF578C)
 
 val green = Color(0xFF25B47E)
+
+val textBlack = Color(0xFF171717)
+val textWhite = Color(0xFFEBEBEB)
```

Comment:
This color that is only used in the AptoideLogo is different from the one in design.